modules/okhttp-client/src/main/java/com/spotify/apollo/http/client/HttpClientDecoratorProvider.java

```diff
@@ -21,12 +21,21 @@
 
 import com.spotify.apollo.environment.ClientDecorator;
 
+import com.google.inject.Inject;
+
 import javax.inject.Provider;
 
 class HttpClientDecoratorProvider implements Provider<ClientDecorator> {
 
+  private final HttpClient httpClient;
+
+  @Inject
+  HttpClientDecoratorProvider(final HttpClient httpClient) {
+    this.httpClient = httpClient;
+  }
```

Comment:
Can't this class be dropped all together now that HttpClientDecorator has an injectable constructor?